src/Carbon/Carbon.php

```diff
@@ -1576,6 +1576,16 @@ public function isLeapYear()
     }
 
     /**
+     * Determines if the instance is a long year
+     *
+     * @return bool
+     */
+    public function isLongYear()
+    {
+        return static::create($this->year, 12, 28, 0, 0, 0, $this->tz)->weekOfYear === 53;
```

Comment:
Ok, I'll make that change.

Here's a source that defines a "long year", would you like it in there also? https://en.wikipedia.org/wiki/ISO_week_date#Weeks_per_year